content/applications/productivity/knowledge/articles_editing.rst

```diff
@@ -0,0 +1,265 @@
+================
+Articles editing
+================
+
+Add and style content
+=====================
+
+To start adding content, click anywhere on the page. Your cursor is automatically set to write the
+article's first-level header. Once you are done writing the title, press **enter** on your keyboard
+to move to the next line.
+
+.. image:: articles_editing/ui.png
+   :align: center
+   :alt: knowledge's user interface
+
+.. tip::
+   To automatically match your :abbr:`h1 (First-level header)` title and the article's name, click
+   on :guilabel:`Untitled` on the left side of the top bar. Doing so synchronizes the
+   *Untitled article* and the header.
+
+.. note:: This method only works one time. To modify your article's name, later on, you have to
+   change it manually. To do so, select the article, click on the name on the top bar and proceed to
+   the modification.
+
+Text editor
+-----------
+
+To stylize the text, you need to open the text editor. To make the editor appear, select the text to
+format.
+
+Then, you can:
+
+- Change the style by clicking on :guilabel:`Normal`. Doing so opens a dropdown menu with multiple
+  styles to choose from (:guilabel:`Normal, Code, Header 1 → Header 6, Quote`);
+- Format the text. Click :guilabel:`B` to put it in bold, :guilabel:`/` to put it in italic,
+  :guilabel:`U` to underline, and :guilabel:`S` to strikethrough);
+- Change the font color by clicking on :guilabel:`A` or the background color by clicking on the
+  **pencil** icon:
+
+  - To choose from a predefined theme color, click :guilabel:`Theme` and select the desired color.
+  - To customize, click :guilabel:`Solid` and define a color using the wheel, by typing its hex
+    code, or its RGBA values.
+  - To use a gradient, click :guilabel:`Gradient`, choose a predefined gradient or click
+    :guilabel:`Custom` to create a personalized gradient.
+
+- To change the text's size, click on the **size number** and select the desired size;
+- Click on the **lists** icons to turn the paragraph into an unordered list, an ordered list, or a
+  checklist;
+- Click the **chain** icon to insert or edit an URL link.
+
+.. image:: articles_editing/style-and-colors.png
+   :align: center
+   :alt: Text editor's toolbox
+
+To format a whole paragraph, type `/` anywhere in the text. Doing so opens the
+:doc:`powerbox </developer/reference/frontend/odoo_editor>`, which allows:
```

Comment:
Not sure that linking the dev doc (powerbox) is a good idea here, as the doc isn't going to be helpful to readers.

(also later in this doc)